Gemfile.lock

```diff
@@ -86,4 +86,4 @@ RUBY VERSION
    ruby 2.7.3p183
 
 BUNDLED WITH
-   2.2.20
+   2.1.4
```

Comment:
I don't know that we want to downgrade this version, but I suppose it shouldn't hurt anything.